cli/src/main.rs

```diff
@@ -33,9 +33,16 @@ static DB: LazyLock<SqliteDb> = std::sync::LazyLock::new(|| {
 });
 
 fn get_signers_with_defaults(private_keys: Option<Vec<String>>) -> Vec<PrivateKeySigner> {
-    if private_keys.is_none() {
-        println!("No private keys provided. Using default private keys.");
+    if private_keys.is_some() {
```

Comment:
good idea, I'd just change the syntax a little.
This is a good spot to use `if let` unwrapping:
```rs
if let Some(private_keys) = private_keys {
    // use provided private_keys
} else {
    // use default keys
}
```